src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -84,24 +126,65 @@ public static int getNumberOfOccurrences(String[] array, String value) {
      * @return array with identical contents excluding values of `value`
      */ // TODO
     public static String[] removeValue(String[] array, String valueToRemove) {
-        return null;
+
+        int numValuesToRemove = 0;
```

Comment:
Redundant code block...
`numValuesToRemove` can be derived from `getNumberOfOccurrences`
```java
int numValuesToRemove = getNumberOfOccurrences(array, valueToRemove);
```